tests/ha/ha_cluster_init.pm

```diff
@@ -0,0 +1,39 @@
+use base "hacluster";
+use testapi;
+
+sub run() {
+   script_run "systemctl restart wicked.service", 60; #update dynamic DNS just in case
+   script_run "ha-cluster-init"; 
+   assert_screen "ha-cluster-init-watchdog-warning";
+   send_key "ctrl-c";
+   script_run "echo softdog > /etc/modules-load.d/softdog.conf";
+   script_run "systemctl restart systemd-modules-load.service";
+   script_run "echo \"softdog=`lsmod | grep softdog | wc -l`\" > /dev/$serialdev";
+   wait_serial("softdog=1"); #softdog is enabled
```

Comment:
wait_serial is never a fatal failure, so I'd recommend adding a `|| die "softdog wasnt enabled"` to it